examples/3zone_toy_stochastic_PySP/rhosetter_FS_only.py

```diff
@@ -22,10 +22,10 @@
 optimizations.
 
 """
-from .rhosetter import set_rho_values
+from rhosetter import set_rho_values
```

Comment:
All your other edits look great!
I personally prefer the import style of `.uninstalled_module` for pulling in python files from the same directory that aren't installed anywhere. Whenever I see an `import foo` in python, I tend to assume that foo is some package installed on the system rather than a file in the working directory. Dunno if that's just my personal preference or a wide-spread practice.

Reply:
I removed the relative import and left it as a global import because Python 3 does not support implicit relative imports; you must specify the precise package from where the module will be read (check this [Quora post](https://www.quora.com/Why-does-Python-3-not-allow-relative-imports)). Thus, this solution provides compatibility for both Python 2 and 3.

Maybe you can add a comment specifying that the module is actually another script located in the same folder and not an installed package?

Reply:
Ah. Sounds like I have more learning to do about python 3. Yeah, a comment makes sense in that case.

In general (for other files), this probably means it's extra important to follow to the convention of separating imports into built-in packages, 3rd party packages, and modules/packages from our own codebase.

Reply:
Strangely, there are two places where I can comment on this commit: one in the pull request [(here)](https://github.com/switch-model/switch/pull/115/commits/121fb879abfeca37386cd3c1f4f812a4ca80967c) and one on the commit itself in our repository [(here)](https://github.com/switch-model/switch/commit/121fb879abfeca37386cd3c1f4f812a4ca80967c). Guess this is the better place for it.

Do you happen to know how `rhosetter.py` is found in this case? I tried moving both rhosetter files into a `setters` subdirectory, then using `--rho-cfgfile=setters/rhosetter_FS_only.py`. This worked fine, but I wouldn't actually have expected it to. 

If I break into the code just below this comment, I find that `sys.path` includes `.` (the main directory) but not `''` as defined by [the docs](https://docs.python.org/3/library/sys.html#sys.path) and found from `switch test`, and also not `setters`. Nevertheless, when `from rhosetter import set_rho_values` runs, it manages to find rhosetter.py inside the `setters` directory. I wouldn't expect that to happen.

It looks like `six` (used by Pyomo) is patching the import mechanism, because if I step into the `from rhosetter import set_rho_values` statement, I end up in the `find_module()` function of `six.py`. I didn't follow it all the way through, but maybe `six` is patching Python to support Python-2-style implicit relative imports? That wouldn't be a great situation for us, because this code will break if Pyomo stops using `six`. So maybe the comment should be "rhosetter.py must be in the Python path, e.g. in the directory that is active when `runph` is called."

For what it's worth, I rely fairly often on the fact that the current working directory is at the top of the search path when you call `switch` (or probably `runph`). I find this is a natural way to include custom modules in a particular model. i.e., I create a main directory for a study (a set of scenarios), and I put modules.txt, options.txt, scenarios.txt and any custom modules at the top level. Then I put custom inputs directories for various scenarios inside that (e.g., inputs/fast_rps, inputs/no_wind, etc.). These inputs directories don't have a modules.txt (they share the main one in the top directory), but the scenario definitions in scenarios.txt may use --include-module[s] or --exclude-module[s] to adjust the module list a bit. Then, when I run `switch solve` or `switch solve-scenarios --scenario x`, etc., in the top-level directory, it can automatically find any custom modules that are stored there. Alternatively, if users have a custom modules that they reuse across different studies, they can put them in some other directory and create a .pth file in site-packages that points to that directory (I haven't done that, but I could see it happening).

Reply:
Sorry, following on @josiahjohnston's comment below, I just took another look at `sys.path` from inside `rhosetter_FS_only`, and `setters` is actually there, right at the top (I missed it because it blended in to the line above, and I assumed '.' was the first entry). So what seems to be happening is simply that `runph` adds the location of the `rhosetter_FS_only.py` to the path before importing that module. So as long as `rhosetter.py` is in the same location, it should also load successfully. This is a little brittle (what if `runph` adopts some other technique for importing modules from arbitrary locations?), but it is an attractive behavior, so we could just live with it for now and leave the comment as-is.